tdrs-frontend/src/actions/auth.js

```diff
@@ -35,20 +35,34 @@ export const CLEAR_AUTH = 'CLEAR_AUTH'
  * the authentication data in the Redux store is cleared, and the user
  * is considered 'logged out', and can no longer access private routes.
  */
+
 export const fetchAuth = () => async (dispatch) => {
   dispatch({ type: FETCH_AUTH })
   try {
     const URL = `${process.env.REACT_APP_BACKEND_URL}/auth_check`
+    const instance = utils.axiosInstance
     const {
-      data: { user },
-    } = await axios.get(URL, {
+      data: { user, csrf },
+    } = await instance.get(URL, {
       withCredentials: true,
     })
+
+    instance.defaults.headers['X-CSRFToken'] = csrf
+
     if (user) {
       dispatch({ type: SET_AUTH, payload: { user } })
     } else {
       dispatch({ type: CLEAR_AUTH })
     }
+    // cookieJar.getCookies(URL, (err, cookies) => {
```

Comment:
Should remove this commented code, we thought we could access the cookies via another route to fix the csrf issue, this turned out to not be true.